### PR TITLE
Show AWS network tags when adding a host

### PR DESCRIPTION
**Pull request by loganhz**
https://github.com/rancher/rancher/issues/5029

### CODE REVIEW

File: app/components/machine/driver-amazonec2/template.hbs [hunk 1]
```diff
@@ -96,13 +96,20 @@
               {{#if choice.isVpc}}
                 <hr class="mt-20 mb-20" />
                 <div class="radio">
-                  <label class="acc-label pb-0">{{radio-button selection=selectedSubnet value=choice.value}} {{choice.label}}</label>
+                  <label class="acc-label pb-0">{{radio-button selection=selectedSubnet value=choice.value}} {{choice.label}}
+                    {{#if choice.tags.length}}
+                      <i class="icon icon-tag"></i> {{choice.tags}}
```
Comment (vincent99) [inline app/components/machine/driver-amazonec2/template.hbs]: Can you wrap these in <span class="tag bg-info">...</span> so they stand out

Comment (loganhz) [reply app/components/machine/driver-amazonec2/template.hbs]: Thanks Vince. Updated.
Do you mean something like this ?

Comment (vincent99) [reply app/components/machine/driver-amazonec2/template.hbs]: Sorry GitHub ate the tag... `<span class="tag bg-info">...</span>`

Comment (loganhz) [reply app/components/machine/driver-amazonec2/template.hbs]: Thanks. Please let me know if I need to change something else

**Pull request merged**